Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1452,7 +1452,13 @@ eval (GetQuotesList (SelectListRes resp)) state = do
               let filteredQuoteList = filter (\a -> length (filter (\b -> a.id == b.id )state.data.quoteListModelState) == 0 ) selectedQuotes
               let removeExpired = filter (\a -> a.seconds > 0) filteredQuoteList
               _ <- pure $ spy "quotes" filteredQuoteList
-              let newState = state{data{quoteListModelState = state.data.quoteListModelState <> removeExpired },props{isSearchLocation = NoView, isSource = Nothing,currentStage = QuoteList}}
+              let quoteListModelState = state.data.quoteListModelState <> removeExpired
```

Comment:
@ashishsingh101 can you add a check for the notification call back that we are getting from native side. Don't exit if the notificationtype == "DRIVER_QUOTE_INCOMING".

make the following modification 

eval (NotificationListener notificationType) state = do
  _ <- pure $ printLog "storeCallBackCustomer notificationType" notificationType
  case notificationType of 
    "DRIVER_QUOTE_INCOMING" -> continue state 
    _ -> exit $ NotificationHandler notificationType state { props { callbackInitiated = false}}